app/src/main/java/com/brainwallet/wallet/BRWalletManager.java

```diff
@@ -514,9 +514,9 @@ public void initWallet(final Context ctx) {
                 String firstAddress = BRWalletManager.getFirstAddress(pubkeyEncoded);
                 BRSharedPrefs.putFirstAddress(ctx, firstAddress);
                 FeeManager feeManager = FeeManager.getInstance();
-                if (feeManager.isRegularFee()) {
-                    feeManager.updateFeePerKb(ctx);
```

Comment:
Very key...Thanks @andhikayuana . This means peoples transactions will go through more quickly...the cost is basically the same.